CHANGELOG.md

```diff
@@ -13,7 +13,11 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
 
 * Added support for adding two nullable columns.
 
-* Addded support for unsigned types in MySQL.
+* Added support for unsigned types in MySQL.
```

Comment:
Glad I'm not the only one. I need to automagically enable spell checking for .md files to avoid things like that in the future ^^